src/frontend/components/Button/index.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { wrapInRouter } from 'utils/tests/router';
+
+import { Button } from '.';
+
+describe('<Button />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('Renders the Button component', () => {
+    render(wrapInIntlProvider(wrapInRouter(<Button label="Button" />)));
+
+    screen.getByRole('button', { name: 'Button' });
+  });
```

Comment:
Please see my comments for src/frontend/components/Badge/index.spec.tsx
I think we should have more tests on this component, regarding all props available.